modules/websearch/lib/websearch_templates.py

```diff
@@ -167,6 +168,9 @@ class Template:
         'wl': (int, CFG_WEBSEARCH_WILDCARD_LIMIT),
         }
 
+    if CFG_WEBSEARCH_OBELIX_REDIS:
+        search_results_default_urlargd['rm'] = (str, "rrm")
```

Comment:
But we want is as default so many places? I guess it will be a lot of work to maintain?

This was the simplest approach I could find to set it as default for the "main search"?